docs/Style.thy

```diff
@@ -0,0 +1,182 @@
+(*
+ * Copyright 2021, Data61, CSIRO
+ *
+ * This software may be distributed and modified according to the terms of
+ * the BSD 2-Clause license. Note that NO WARRANTY is provided.
+ * See "LICENSE_BSD2.txt" for details.
+ *
+ * @TAG(DATA61_BSD)
+ *)
+
+chapter \<open>An Isabelle Syntax Style Guide\<close>
+
+theory Style
+  imports Main
+begin
+
+(* This style guide is heavily influenced by Gerwin's Style Guide for Isabelle/HOL. See parts 1
```

Comment:
We should make these `text` comments instead of actual comments. That way, they are rendered readable on GitHub and they do show up in html/pdf if we produce that.